generator/resources/ec2_linux_test_matrix.json

```diff
@@ -21,6 +21,17 @@
     "binaryName": "amazon-cloudwatch-agent.deb",
     "family": "linux"
   },
+  {
+    "os": "ubuntu-23",
+    "username": "ubuntu",
+    "instanceType":"t3a.medium",
```

Comment:
I think we want to favor using arm or amd.

Reply:
Maybe we should have some tests against intel amd64 since we only currently have amd64 and arm64. Intel is t3.medium. a indicates amd, g indicates AWS built hardware

Reply:
this is running amd64, I looked at the existing Ubuntu tests, and they use the the same instance type